glidertest/plots.py

```diff
@@ -914,3 +914,64 @@ def plot_combined_velocity_profiles(ds_out_dives: xr.Dataset, ds_out_climbs: xr.
         ax.legend()
         plt.show()
         return fig, ax
+
+    def plot_hysteresis(ds, var='DOXY', v_res=1, perct_err=2, ax=None):
```

Comment:
you have this whole block indented, that's why the import doesn't work and notebooks fail